x-pack/plugins/lists/server/config.ts

```diff
@@ -9,6 +9,7 @@ import { TypeOf, schema } from '@kbn/config-schema';
 export const ConfigSchema = schema.object({
   enabled: schema.boolean({ defaultValue: true }),
   importBufferSize: schema.number({ defaultValue: 1000, min: 1 }),
+  importTimeout: schema.number({ defaultValue: 300000, max: 6000000, min: 120000 }), // Default value is 300000 (3 minutes), maximum of 6000000 (100 minutes), minimum of 120000 (2 minutes)
```

Comment:
there is `schema.duration` type available that allows human-readable format. Although, it doesn't support `min`, `max` options. (can be easily added?)
```yml
lists. importTimeout: 120s
```
```js
schema.duration({
  defaultValue: '30s',
  validate: (value: Duration) => {
    if (2 > value.asMinutes()){ 
      throw new Error('duration cannot be less than 2 minutes')
    }
   //...
})
```

Reply:
Oh that's an interesting validator. I can give this type a try and see if I utilize it instead of number. That should make things more user configuration friendly.